puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-infra-jenkins-jobs-update.yml

```diff
@@ -0,0 +1,12 @@
+---
+- job:
+    name: 'foreman-infra-jenkins-jobs-update'
+    project-type: 'pipeline'
+    sandbox: true
+    scm:
+      - foreman-infra
+    triggers:
+      - pollscm: ''
+    dsl:
+      !include-raw:
+        pipelines/infra/updateJobs.groovy
```

Comment:
would that need to be

```
- pipelines/infra/updateJobs.groovy
```

or is jjb not too picky about lists?

Reply:
I've built it from the yaml, so I don't think it's too picky.